documentation/book/master.adoc

```diff
@@ -11,6 +11,8 @@ include::getting-started.adoc[leveloffset=+1]
 
 include::assembly-deployment-configuration.adoc[leveloffset=+1]
 
+include::assembly-deployment-configuration-kafka-mirror-maker.adoc[leveloffset=+1]
```

Comment:
What is the reason to include this as chapter 4 and not as 3.4 as it used to be? I think I would prefer it to be 3.4 because:
* That is where all the other configuration chapters are
* The _Customizing Deployments_ section applies to all of them. This is now not clear.

Reply:
We can move it back, but we lose the individual properties links in the TOC. 

The high level assembly says very little (and it doesn't mention the Bridge), it just adds another level.  

I'm not sure it's clear that the Customizing Deployments section applies to all components wherever the other content sits. 

An alternative would be to break up the assembly and have separate configuration sections for each component (that we work to reduce) and a Customizing Deployment section that makes it clear what it applies to. Then we wrap this into a Configuration Guide (eventually)

Reply:
Well, I think we should have this in sync somehow ... either we need to move them all. Or we should keep them all inside that chapter.

Reply:
Okay. I've moved it back into the config assembly for now.